articles/clients/client-types-and-settings.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Describes the differences between the Auth0 Client types and what they mean when it comes to settings
+---
+
+# Settings for Client Types
```

Comment:
I would just name this `Client Settings`, the way it is now it's like this `Client Types` is some kind of object.